packages/tools/client-debugger/client-debugger-view/src/components/AudienceHistoryTable.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import {
+	Avatar,
+	TableBody,
+	TableCell,
+	TableRow,
+	Table,
+	TableHeader,
+	TableHeaderCell,
+} from "@fluentui/react-components";
+import { Clock20Regular } from "@fluentui/react-icons";
+import { FilteredAudienceHistoryData } from "./AudienceView";
+
+/**
+ * Input for {@link AudienceHistoryTable}
+ */
+export interface AudienceHistoryTableProps {
+	audienceHistoryItems: FilteredAudienceHistoryData[];
+}
+
+/**
+ * Render audience history in {@link AudienceView}
+ */
+export function AudienceHistoryTable(props: AudienceHistoryTableProps): React.ReactElement {
+	const { audienceHistoryItems } = props;
+
+	// Columns for rendering audience history
+	const audienceHistoryColumns = [
+		{ columnKey: "clientId", label: "ClientId" },
+		{ columnKey: "time", label: "Time" },
+	];
+
+	return (
+		<Table size="small" aria-label="Audience history table">
+			<TableHeader>
+				<TableRow>
+					{audienceHistoryColumns.map((column, columnIndex) => (
+						<TableHeaderCell key={columnIndex}>
+							{column.columnKey === "clientId" && <Avatar />}
+							{column.columnKey === "time" && <Clock20Regular />}
+							{column.label}
+						</TableHeaderCell>
+					))}
+				</TableRow>
+			</TableHeader>
+			<TableBody>
+				{audienceHistoryItems.map((item, itemIndex) => (
+					<TableRow
+						key={itemIndex}
+						style={{
+							backgroundColor: item.changeKind === "added" ? "#90ee90" : "#FF7377",
```

Comment:
This should be using the `tokens` object Rohan showed us for the colors, right?

Reply:
Resolved.